pkgs/applications/networking/p2p/zeronet/default.nix

```diff
@@ -12,24 +13,24 @@ python2Packages.buildPythonApplication rec {
   };
 
   propagatedBuildInputs = with python2Packages; [ msgpack gevent ];
-
-  format = "other";
-
   buildPhase = "${python2Packages.python.interpreter} -O -m compileall .";
 
   installPhase = ''
     mkdir -p $out/share
     cp -r plugins src tools *.py $out/share/
   '';
 
-  # Wrap the main executable and set the log and data dir to something out of
-  # the store
+  # Wrap the main executable and set the log and data dir to something out of the store
   postFixup = ''
     makeWrapper "$out/share/zeronet.py" "$out/bin/zeronet" \
-        --set PYTHONPATH "$PYTHONPATH" \
-        --set PATH ${python2Packages.python}/bin \
-        --add-flags "--log_dir \$HOME/.local/share/zeronet/logs" \
-        --add-flags "--data_dir \$HOME/.local/share/zeronet"
+      --set PYTHONPATH "$PYTHONPATH" \
+      --set PATH ${python2Packages.python}/bin \
+      --add-flags "--log_dir \$HOME/.local/share/zeronet/logs" \
+      --add-flags "--data_dir \$HOME/.local/share/zeronet"
```

Comment:
I don't think this binary should have been wrapped like this in the first place. Nix packages shouldn't pass convenience arguments, that's up to the user. If anything I'd argue to remove these arguments for the main binary and not have the `zeronet_service` wrapper, makes much more sense.

Reply:
Unfortunately, zeronet will not start without log_dir and data_dir set, so I think I understand why @fgaz made this a wrapper. Not sure about PYTHONPATH or PATH, though, but I remember having problems calling the .py directly without the wrapper.

Reply:
Nix packages don't need to provide a binary that runs *without any arguments*, they just need to provide the binary that gets distributed, optionally wrapped with some env vars to make it work under Nix. Passing the right arguments will be the job of either the user or some wrapper higher up.

Reply:
Okay, I'll see if I can't get rid of the wrappers altogether, but I may still have to wrap the PYTHONPATH and maaaaaybe the PATH. If log_dir and data_dir aren't a problem, then I can easily get rid of the zeronet_service wrapper too.